lib/matplotlib/mpl-data/stylelib/_internal_test_patch.mplstyle

```diff
@@ -0,0 +1,4 @@
+# This patch should go on top of the "classic" style and exists solely to avoid
```

Comment:
I would leave "classic" in the name. Maybe "_classic_test_patch.mplstyle". This makes it clear that it's only intended for the classic style.

~~~
style = ["classic", "_classic_test_patch"]
~~~

reads quite consistent.

"internal" is not adding much as we already have a underscore prefix.

Reply:
ok, renamed to _classic_test_patch